cfgov/unprocessed/apps/analytics-gtm/js/bah-closing-disclosure-listeners.js

```diff
@@ -4,6 +4,6 @@ import bahExplainers from './bah-explainers';
 
 const OAHCDAnalytics = ( () => {
 
-  bahExplainers( 'OAH Closing Disclosure Interaction' );
+  bahExplainers( 'OAH Closing Disclosure' );
```

Comment:
`Interaction` is added at https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/apps/analytics-gtm/js/bah-explainers.js#L13